stats/collector.go

```diff
@@ -206,6 +213,20 @@ func (s *BufferedCollector) CollectMetadata(accountMetadata *Metadata) {
 	}
 }
 
+func (s *BufferedCollector) collectHeartbeat(ctx context.Context) {
+	// consistent id for all heartbeats of this run
+	const idLength = 16
+	uniqueRunID := randstr.Hex(idLength)
+	for {
+		select {
+		case <-time.After(heartbeatInterval):
+			s.CollectEvent("heartbeat", uniqueRunID)
```

Comment:
I would add this event as class = "global" (which already exists for events that aren't tied to user actions) and action = "heartbeat" which in any case should remain a static constant to allow filtering by it.

Reply:
Done.